content/en/security/application_security/setup/ruby/troubleshooting.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Troubleshooting Ruby App and API Protection
+---
+
+## Common Issues
+
+### No security signals appearing
+
+1. **Verify Agent version**
+   - Ensure you're running Datadog Agent v7.41.1 or higher
+   - Check Agent status: `datadog-agent status`
+
+2. **Check Ruby tracer version**
+   - Confirm you're using Ruby tracer v1.9.0 or higher
+
+3. **Verify configuration or environment variables**
+
+#### When using auto-instrumentation
+
+- Ensure that `DD_APPSEC_ENABLED` is set to `true`
+- Check `DD_SERVICE` and `DD_ENV` are properly configured
+- Set `DD_AGENT_HOST` and `DD_AGENT_PORT` if agent is running on a different host or port
+- Check that `DD_SERVICE` and `DD_ENV` are properly configured
+- Verify `DD_APM_ENABLED=true` if using APM features
```

Comment:
Is the mention of this env var here expected? There is no mention of this env var in AAP, the mentioned one is `DD_APM_TRACING_ENABLED`

Reply:
good catch, fixed in [cf8287f](https://github.com/DataDog/documentation/pull/30252/commits/cf8287fd4e7c8053d9519eed4c7dde2f2ed6acda)